demos

```diff
@@ -71,17 +71,17 @@ click on the name of the output file:
 
 16. HTML slide shows:
 
-@   pandoc -s --mathml -i -t dzslides @@SLIDES@@ -o @@example16a.html@@
-@   pandoc -s --webtex -i -t slidy @@SLIDES@@ -o @@example16b.html@@
-@   pandoc -s --mathjax -i -t revealjs @@SLIDES@@ -o @@example16d.html@@
+@   pandoc -s --mathml -i -t dzslides @@SLIDES.txt@@ -o @@example16a.html@@
+@   pandoc -s --webtex -i -t slidy @@SLIDES.tx@@ -o @@example16b.html@@
+@   pandoc -s --mathjax -i -t revealjs @@SLIDES.txt@@ -o @@example16d.html@@
 
 17. TeX math in HTML:
 
-@   pandoc @@math.text@@ -s -o @@mathDefault.html@@
-@   pandoc @@math.text@@ -s --mathml  -o @@mathMathML.html@@
-@   pandoc @@math.text@@ -s --webtex  -o @@mathWebTeX.html@@
-@   pandoc @@math.text@@ -s --mathjax -o @@mathMathJax.html@@
-@   pandoc @@math.text@@ -s --katex   -o @@mathKaTeX.html@@
+@   pandoc @@math.tex@@ -s -o @@mathDefault.html@@
+@   pandoc @@math.tex@@ -s --mathml  -o @@mathMathML.html@@
+@   pandoc @@math.tex@@ -s --webtex  -o @@mathWebTeX.html@@
+@   pandoc @@math.tex@@ -s --mathjax -o @@mathMathJax.html@@
+@   pandoc @@math.tex@@ -s --katex   -o @@mathKaTeX.html@@
```

Comment:
Note that this file is a markdown file, not a tex file.
The first line is a pandoc title block.
Changing the extension to tex would cause pandoc to parse it as a LaTeX rather than an (extended) Markdown file.